### PR TITLE
Ind backend validation.

### PR DESCRIPTION
**Pull request by huxuan**
- Add `order` paramter in `postIndividual()`.
- Hack `errmsg()` with custom information as `err_custom_msg()`.
- Individual validation should work now.
- The Database structure is also changed by add `order` column in people.

Plz HELP me to test all possible situations, please.

__We should introduce unit test for our project next time!__

### CODE REVIEW

**Comment by huxuan** [comment]
For the `order` column in `people` table, I think we also need it in fetch. As we should list all people by their original order, otherwise it may look a little wired. @JeldorPKU

**Comment by huxuan** [comment]
Hurray! This is the #100 one!

File: application/controllers/registration.php [hunk 1]
```diff
@@ -46,10 +46,149 @@ public function individual() {
...
+                    )));
+                }
+                // meal16
+                if (!array_key_exists($item_post['meal16'], $GLOBALS['JUDGE'])) {
+                    exit(err_custom_msg('1070', array(
+                        'order' => $item_post['order'] + 1,
+                    )));
+                }
+                // meal17
+                if (!array_key_exists($item_post['meal17'], $GLOBALS['JUDGE'])) {
```
Comment (JeldorPKU) [inline application/controllers/registration.php]: 这里加一个条件，参赛的必须吃17日的午饭。

**Comment by JeldorPKU** [comment]
参赛的人是可以只参加团体赛的，这个条件好像没加。我测试的时候如果只参加团体赛会返回错误。

File: application/controllers/registration.php (hunk 1 above)
Comment (huxuan) [inline application/controllers/registration.php]: Done.

**Pull request merged**